controller.java

```diff
@@ -5,10 +5,10 @@ public static void main(String[] args) {
         int num1 = 10;
         int num2 = 20;
 
-        // Perform addition
-        int sum = num1 + num2;
-
+int summation = math.add(number2, num1);
+        int summation = math.add(num2, num1);
+        System.out.println("xofsz");
```

Comment:
change variable name to total
```suggestion
    int total = math.add(number2, num1);
```